### PR TITLE
Added 3.12.0 to releases

### PR DESCRIPTION
**Pull request by Ben-Henshall**
Hello! Even our app needs a check-up from time to time. That's why we've been busy making some enhancements, so you can get the most from it.
In this update, we've improved our prescription functionality. It's now easier for you to collect your medication or arrange medication delivery of your private prescriptions.

Thank you for using Babylon! If you have any suggestions on how we can improve our service, please let us know at support@babylonhealth.com

### CODE REVIEW

File: release.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ The release process starts when the first build is provided to QA and ends when
 
 | Version | Release Engineer(s)  | QA effort   | Engineering effort          | Total effort  | Cut-off date  | Release date  |
 |---------|----------------------|-------------|-----------------------------|---------------|---------------|---------------|
+| 3.12.0                   | Ben Henshall<br> Danilo Aliberti | Automated: `07h16m`<br>Manual: `22h`| `MON-4225: 3h`| Total: **32h16m** | 18.03.2019 | 21.03.2019 |
```
Comment (JasonDobo) [inline release.md]: Do we not include automation time?

Comment (Ben-Henshall) [reply release.md]: It's already there:
>Automated: `07h16m`

Comment (Ben-Henshall) [reply release.md]: The automation time is included in the total time yes - what exactly do you want to mention?

Comment (JasonDobo) [reply release.md]: Forget what I said, I'm going blind apparently

**Pull request merged**